lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
>> On Mon, Apr 04 2016 16:09, Lionel Henry wrote:

>> The long name ess-format-eval-command is not very appealing to me because 'format' reads as verb in there.

> That's intended, it does format a function to send to the inferior based on a
> bunch of parameters.

I see. How about more meaningful ess-build-eval-command command then. In pure
functional tradition it would be sufficient to have ess-eval-command, but it's
too confusing.

>> We use terms source, load and eval somewhat interchangeably.
> Load is about a file, eval about a command.

Internally it comes down to the same thing - sourcing files.

I think it would be cleaner to deal with formatting issues in one "format"
command. Let's see what are other IDE's are calling it.

> Where things get blurry is that we load files through eval for tramp
> performance... I was confused for a long time by this.

I don't remember the details but this is happening only when the file is not on
a remote. It should not happen when file and process are on the same machine.

  Vitalie